binaries/build.py

```diff
@@ -23,26 +41,58 @@ def build():
     print(f"## In directory: {os.getcwd()} | Executing command: {create_wheel_cmd}")
     ma_build_exit_code = os.system(create_wheel_cmd)
 
+    os.chdir(REPO_ROOT)
+    
+    # Build model archiver wheel
+    os.chdir("workflow-archiver")
+    print(f"## In directory: {os.getcwd()} | Executing command: {create_wheel_cmd}")
+    wa_build_exit_code = os.system(create_wheel_cmd)
+
     os.chdir(REPO_ROOT)
 
     ts_wheel_path = glob.glob(os.path.join(REPO_ROOT, "dist", "*.whl"))[0]
     ma_wheel_path = glob.glob(os.path.join(REPO_ROOT, "model-archiver", "dist", "*.whl"))[0]
+    wa_wheel_path = glob.glob(os.path.join(REPO_ROOT, "workflow-archiver", "dist", "*.whl"))[0]
     print(f"## TorchServe wheel location: {ts_wheel_path}")
     print(f"## Model archiver wheel location: {ma_wheel_path}")
+    print(f"## Workflow archiver wheel location: {ma_wheel_path}")
```

Comment:
Please update readme about model archiver and workflow archiver binary location.